.pre-commit-config.yaml

```
@@ -70,7 +70,7 @@ repos:
           - tomli
 
   - repo: https://github.com/pycqa/isort
-    rev: 5.11.4
+    rev: 5.11.5
```

5.11.5 was for projects that still support Python 3.7, we should be able to use 5.12.0. See also PyCQA/isort#2083
```suggestion
    rev: 5.12.0
```